src/requirements.txt

```diff
@@ -1,16 +1,32 @@
+blis
+catalogue
 convokit
+cycler
+cymem
+ftfy
 gensim
 imblearn
+importlib_metadata
+kiwisolver
 markdown
 matplotlib
+murmurhash
 nltk
 numpy
 pandas
+plac
+preshed
+pytz
+requests
 scipy
 sklearn
 spacy==2.2.4
+srsly
 textblob
+thinc
+threadpoolctl
+wasabi
 wordfreq
-requests
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.0/en_core_web_sm-2.2.0.tar.gz#egg=en_core_web_sm==2.2.0
+zipp
 https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.2.0/en_core_web_md-2.2.0.tar.gz#egg=en_core_web_md==2.2.0
+https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.2.0/en_core_web_sm-2.2.0.tar.gz#egg=en_core_web_sm==2.2.0
```

Comment:
still confusing to me why I'm seeing changes that were present in prior PR. maybe your branch is stale? can you updated it to head?

Reply:
Outdated.